src/app/frontend/replicasetlist/logsmenu_controller.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Controller for the logs menu view.
+ *
+ * @final
+ */
+export default class LogsMenuController {
```

Comment:
You can verify that openMenu method does `this.replicaSetPodsList = [];`. It is a logic and should be tested. It should be easy :)